frontend/packages/integration-tests-cypress/views/modal.ts

```diff
@@ -0,0 +1,10 @@
+import { submitButton } from './form';
+
+export const modal = {
+  shouldBeOpened: () => cy.byLegacyTestID('modal-cancel-action').should('be.visible'),
+  typeInField: (id: string, value: string) => cy.get(`[data-test="${id}"]`).type(value),
```

Comment:
I think just inlining this is better:

```ts
cy.byTestID('my-input').type('foo');
```

It's more clear what's happening, and it's not obvious that `id` here refers to a test ID

Reply:
agree, fixed.